source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -0,0 +1,161 @@
+.. _node-fundamentals-distinct:
+
+========================
+Retrieve Distinct Values
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Use the ``distinct()`` method to retrieve all distinct values for a specified field 
+across a collection.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+Follow the instructions in the examples below to insert data into a collection and
+return the unique values of a given field. Consider the following collection of 
+documents that describe restaurants: 
+
+.. code-block:: json
+
+   [
+      { "_id": 1, "restaurant": "White Bear", "borough": "Queens", "cuisine": "Chinese" },
+      { "_id": 2, "restaurant": "Via Carota", "borough": "Manhattan", "cuisine": "Italian" },
+      { "_id": 3, "restaurant": "Borgatti's", "borough": "Bronx", "cuisine": "Italian" },
+      { "_id": 4, "restaurant": "Tanoreen", "borough": "Brooklyn", "cuisine": "Middle Eastern" },
+      { "_id": 5, "restaurant": "Äpfel", "borough": "Queens", "cuisine": "German" },
+      { "_id": 6, "restaurant": "Samba Kitchen", "borough": "Manhattan", "cuisine": "Brazilian" },
+   ]
+
+.. include:: /includes/access-cursor-note.rst
+
+Distinct
+--------
+
+Use a document field name as a parameter for the ``distinct()`` method to return a 
+list of the field's unique values. Optionally, you can specify a ``query`` 
+parameter to narrow down your results or an ``options`` parameter to set the 
+string comparison language.
+
+Example
+```````
+
+Some boroughs, like "``Queens``", are represented multiple times in the restaurant 
+documents. However, the following example retrieves only the unique values of the 
+``borough`` field: 
```

Comment:
issue: It might be more obvious to readers to explicitly state that although "Queens" and "Manhattan" are present in many documents, distinct retrieves the unique values of the borough field (only and unique being fairly synonymous).